trie/hexary.py

```diff
@@ -208,20 +208,31 @@ def root_node(self, value):
     #
     # Utils
     #
+    def _set_raw_node(self, raw_node):
+        key, value = self._node_to_db_mapping(raw_node)
+        if key == BLANK_NODE:
+            # skip saving the blank node to DB
+            return BLANK_NODE_HASH
+
+        if value is None:
```

Comment:
Maybe the separation was/is intentional but this seem like a logical place to continue using `elif/else`.

Reply:
Yeah, since the first if block has an early return and the second two blocks are assigning the same two variables ahead of a return, it seemed like a nice logical separation.